test/e2e/secrets_test.go

```diff
@@ -128,6 +129,31 @@ func testUserDataTamper(t *testing.T) {
 	}
 }
 
+// testPrivateKeyChange alters the private key used to SSH into instances and ensures nodes are updated properly
+func testPrivateKeyChange(t *testing.T) {
```

Comment:
s/testPrivateKeyChange/privateKeyChangeTest

Reply:
`testFooBar` seems to be standard naming within our e2e suite

Reply:
I'm following [reconfigurationTest](https://github.com/openshift/windows-machine-config-operator/blob/1ed33ba363613f96840f327419a4739e89abd4a0/test/e2e/reconfigure_test.go#L16)

Reply:
Actually `reconfigurationTest` is not following the Go standard naming.

Reply:
Actually it is a test suite and hence named `reconfigurationTest` which @saifshaikh48 has now correctly named `reconfigurationTestSuite`.